Share The Load/Screens/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ViewController.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 10/01/2021.
+//
+
+import UIKit
+import Firebase
+import RealmSwift
+
+class ViewController: UIViewController {
+    
+    private let getRealmQuestionsButton = UIButton()
+    private let removeFromRealmButton = UIButton()
```

Comment:
Remember the concept of Realm shouldn't be something any other objects are aware of. We designed the PersistenceService public interface with either objects don't need to know where it gets its data from.

Reply:
Are you suggesting that these buttons should be renamed so that they don't include a reference to Realm?

Reply:
FYI - updated in last commit as per your DM on Twitter